spec/controllers/sessions_controller_spec.rb

```diff
@@ -18,9 +18,8 @@
 
   describe 'POST create' do
     context 'with authenticated' do
-      
-      let(:user) { Fabricate(:user) } 
-      before do   
+      before do 
+        user = Fabricate(:user)
```

Comment:
You had it right before. You don't have access to `user` when it is in the `before` block. You should change this spec back to how it was.